airflow/www/templates/appbuilder/navbar.html

```diff
@@ -50,6 +50,13 @@
         {% include 'appbuilder/navbar_menu.html' %}
       </ul>
       <ul class="nav navbar-nav navbar-right">
+        <li class="active">
+          <div style="padding: 20px; cursor: pointer" id="themeToggleButton">
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-moon" viewBox="0 0 16 16" id="themeToggleButton">
```

Comment:
Usually when light/dark is toggled the icon changes from a moon to a sun.

Did the material icons for light_mode and dark_mode not work? I'd prefer to not add a brand new svg if we don't have to.

Reply:
Hello @bbovenzi 
Improved the icon to change when toggling between dark and light mode! Thanks to your suggestion, this feature is a little more SWAG!👍